packages/client/src/components/Layout/MainLayout.tsx

```diff
@@ -27,11 +30,22 @@ type LayoutProps = {
   children: React.ReactNode;
 };
 
-const MainLayout: FC<LayoutProps> = ({ children }) => (
-  <StMainScreen>
-    <Header />
-    <StContent>{children}</StContent>
-  </StMainScreen>
-);
+const MainLayout: FC<LayoutProps> = ({ children }) => {
+  const dispatch = useAppDispatch();
+  const [, , isLoading] = authState();
```

Comment:
можно из authState возвращать объект, а не массив, чтобы получить нужную переменную без костылей):
```suggestion
  const { isLoading } = authState();
```

Reply:
исправил